src/Util/Achievement.h

```diff
@@ -0,0 +1,73 @@
+#pragma once
+
+#include <math/seadVector.h>
+
+namespace al {
+class IUseMessageSystem;
+class LayoutActor;
+class LiveActor;
+}  // namespace al
+
+class AchievementHolder;
+class AchievementNpc;
+class GameDataHolderAccessor;
```

Comment:
I'm not sure if you can pre-declare it, and then use it as non-pointer type. Please check this, by including this header somewhere for testing.

Reply:
Clang does not seem to care when included in another file.